tests/queries/models.py

```diff
@@ -482,6 +482,12 @@ class Meta:
         ordering = ['id']
 
 
+class NullableRectangle(models.Model):
+    id = models.IntegerField(primary_key=True)
+    length = models.IntegerField(null=True)
+    width = models.IntegerField(null=True)
```

Comment:
You should be able to reuse an existing model, e.g. `Number`.

Reply:
Certainly, and when I did simplify the test to reuse that model it revealed a gap in the patch. The original patch doesn't work if `col` is not nullable, as it would have to be for me to use `Number`. I'll take another look.

Reply:
We can add a second nullable column to the `Number` model.